terraform/step_function.tf.json.template.py

```diff
@@ -0,0 +1,70 @@
+import json
+
+from azul import config
+from azul.deployment import aws
+from azul.template import emit
+
+
+def create_lambda_name(basename):
+    return f"arn:aws:lambda:{aws.region_name}:{aws.account}:function:{config.service_name}-{basename}"
```

Comment:
I wonder if this should be moved to `Config.get_lambda_arn(function_name, suffix)`